components/electric/lib/electric/satellite/serialization.ex

```diff
@@ -475,6 +476,11 @@ defmodule Electric.Satellite.Serialization do
   """
   @spec decode_column_value(binary, atom) :: binary
 
+  def decode_column_value(val, :bool) do
+    true = val in ["t", "f"]
+    val
+  end
```

Comment:
For consistency with date PR:
```suggestion
  def decode_column_value(val, :bool) when val in ["t", "f"], do: val
```

Reply:
I'm actually siwtching to doing pattern matching inside the function body in the date PR. There's a likelihood that we'll need a catch-all clause for a particular "electrify dangerously" James has mentioned recently. So it's better for a given known type to proceed into the function body and fail there rather than rely on FunctionClauseError to do the right thing.